demo/admin/src/common/PagesPage.tsx

```diff
@@ -0,0 +1,73 @@
+import { gql } from "@apollo/client";
+import { Field, FinalFormSelect } from "@comet/admin";
+import { AllCategories, createEditPageNode, DocumentInterface, DocumentType, PagesPage as CometPagesPage } from "@comet/cms-admin";
+import { MenuItem } from "@mui/material";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+interface PagesPageProps {
+    allCategories: AllCategories;
+    documentTypes: Record<DocumentType, DocumentInterface>;
+}
+
+const userGroupOptions = [
+    {
+        label: "Show for all",
+        value: "All",
+    },
+    {
+        label: "Show only for Group: User",
+        value: "User",
+    },
+    {
+        label: "Show only for Group: Admin",
+        value: "Admin",
+    },
+];
+
+export const additionalPageTreeNodeFieldsFragment = gql`
+    fragment PageTreeNodeAdditionalFields on PageTreeNode {
+        userGroup
+    }
+`;
+
+const EditPageNode = createEditPageNode({
```

Comment:
I think it would be better to move the EditPageNode to a separate file and using the library's PagesPage in App.tsx instead of creating a separated PagesPage file.

Reply:
true, done 👍